src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -40,6 +40,9 @@
          * to think about number formatting issues.
          */
         Utils.forceJVMLocaleToUSEnglish();
+
+        // Set config factory to know about the configuration options that we have.
+        ConfigUtils.setConfigFactoryVariableDefaults();
```

Comment:
@magicDGS Yeah - this should definitely be customizable.  @lbergelson suggested we find a way to do that above.  I'm still thinking about it - I think a ConfigSettings class or something like that would work.  Ideally it would just search the class path for anything extending the OWNER Config class, then downstream users wouldn't have to think about it (however that would take a long time to search).

It has to be static because of a bug in the OWNER library.  

Briefly, it allows you to specify a Sources annotation with a list of places in which to look for a configuration file.  This Sources annotation supports variables with the syntax ${VARNAME}.  However, if you don't specify VARNAME in either the environment or system properties, OWNER won't know it's a variable and will interpret ${VARNAME} as a raw path instead.  Since this happens when you try to load a config file, we resolve all variables as early as possible (by checking for their existence in system and environment properties).  They can still be overridden with a command-line argument.